spec/unit/repositories_renderer_spec.rb

```diff
@@ -49,6 +51,7 @@
     Enabled: Yes
     Refresh: No
     Priority: 3
+    Type: 
```

Comment:
If a repository was added without specifying the type and `zypper refresh` wasn't run the type is `nil`. I can add something in brackets if this happens.